frontends/election-manager/src/components/election_manager_writein_tally_report.tsx

```diff
@@ -89,7 +89,7 @@ export const ElectionManagerWriteInTallyReport = forwardRef<
     );
 
     return (
-      <TallyReport className="print-only" ref={ref}>
+      <TallyReport ref={ref}>
```

Comment:
is the ref still needed?

Reply:
It's not being used anywhere anymore, should I just get rid of it?

Reply:
Removed it.